.github/workflows/publish.yml

```diff
@@ -22,76 +22,100 @@ concurrency:
   group: deploy
   cancel-in-progress: true
 
-# We need these permissions create new GitHub releases
-permissions: write-all
-
 jobs:
+  deploy_storybook:
+    name: Deploy Storybook
+    uses: ./.github/workflows/deploy-storybook.yml
```

Comment:
**Suggestion:** Should we deploy the storybook after publishing to npm and cdn? 

That way if something fails we'd avoid updating it